openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/SchemaTablesTab.tsx

```diff
@@ -69,6 +75,46 @@ function SchemaTablesTab({
   pagingInfo,
 }: Readonly<SchemaTablesTabProps>) {
   const { t } = useTranslation();
+  const [localTableData, setLocalTableData] = useState<Table[]>([]);
```

Comment:
Let's not use a different state to store the same table data inside the child component. We can use the state from the parent itself.

Reply:
Updated the code.